### PR TITLE
Testing CoffeeScript

### PR DESCRIPTION
**Pull request by malandrina**

### CODE REVIEW

File: book/testing/coffeescript_tests.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+### CoffeeScript Tests
+
+We'll use [Konacha](https://github.com/jfirebaugh/konacha) to write unit tests
```
Comment (malandrina) [inline book/testing/coffeescript_tests.md]: Should we discuss the following items in this introductory paragraph?
1. Why we're writing unit tests and not acceptance tests
2. Konacha vs. Jasmine

File: book/testing/coffeescript_tests.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+for our CoffeeScript classes. Konacha relies on the
+[Mocha](http://visionmedia.github.io/mocha/) test framework and
+[Chai](http://chaijs.com/) assertion library.
+
+First, we add Konacha and Poltergeist to the Gemfile:
```
Comment (malandrina) [inline book/testing/coffeescript_tests.md]: I've been meaning to ask -- why are we using Poltergeist?

Comment (joshuaclayton) [inline book/testing/coffeescript_tests.md]: Poltergeist is what we're using to run the JS suite: https://github.com/thoughtbot/geocoding_on_rails/blob/master/example_app/config/initializers/konacha.rb#L3-L4

File: book/testing/coffeescript_unit_tests.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+` spec/javascripts/current_location_spec.coffee@e9f6739:12,18
+
+Finally, we ensure `CurrentLocation::DEFAULT_LOCATION` returns the expected
+value:
+
+` spec/javascripts/current_location_spec.coffee@e9f6739:20,22
+
+#### Testing `ExampleApp.ReverseGeocoder`
+
+To test `ReverseGeocoder#location`, we'll need to stub requests the external
```
Comment (malandrina) [inline book/testing/coffeescript_unit_tests.md]: Typo: "stub requests to the external geocoding service"

**Pull request merged**